pallets/omnipool/src/benchmarks.rs

```diff
@@ -1 +1,115 @@
+// This file is part of HydraDX-node
 
+// Copyright (C) 2020-2022  Intergalactic, Limited (GIB).
+// SPDX-License-Identifier: Apache-2.0
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#![cfg(feature = "runtime-benchmarks")]
+
+use super::*;
+
+use sp_runtime::FixedU128;
+
+use frame_benchmarking::account;
+use frame_benchmarking::benchmarks;
+use frame_system::RawOrigin;
+use orml_traits::MultiCurrencyExtended;
+
+fn funded_account<T: Config>(name: &'static str, index: u32) -> T::AccountId {
+	let caller: T::AccountId = account(name, index, 1);
+	caller
+}
+
+benchmarks! {
+	 where_clause {  where T::AssetId: From<u32>,
+		T::Balance: From<u128>,
+		T::Currency: MultiCurrencyExtended<T::AccountId, Amount=i128>,
+		T: crate::pallet::Config
+	}
+
+	initialize_pool{
+		let stable_amount: T::Balance = T::Balance::from(1_000_000_000_000_000u128);
+		let native_amount: T::Balance = T::Balance::from(1_000_000_000_000_000u128);
+		let stable_price: FixedU128= FixedU128::from((1,2));
+		let native_price: FixedU128= FixedU128::from(1);
+
+	}: _(RawOrigin::Root, stable_amount, native_amount, stable_price, native_price)
+	verify {
+	}
```

Comment:
nitpick: might be good to verify the stable asset and native asset "arrived" in storage?